src/it/scala/org/ergoplatform/it/KnownNodesSpec.scala

```diff
@@ -0,0 +1,42 @@
+package org.ergoplatform.it
+
+import org.ergoplatform.it.util.IntegrationTest
+import org.scalatest.FreeSpec
+
+import scala.concurrent.Await
+import scala.concurrent.Future.traverse
+import scala.concurrent.duration._
+
+class KnownNodesSpec(nodes: Seq[Node]) extends FreeSpec with IntegrationTest {
+
+  val blocksCount = 5
+  val forkDepth = blocksCount
+
+  s"The third node knows first node" in {
+
+    val node01 = nodes.find(_.settings.scorexSettings.network.nodeName == "node01")
+    val node03 = nodes.find(_.settings.scorexSettings.network.nodeName == "node03")
+
+    val result = node03.map(_.waitForPeers(2).map { peers =>
```

Comment:
`waitForPeers` doesn't include `knownPeers` currently. You can try `/peers/all` call to get all the known peers, or establish link between nodes with `/peers/connect` API, not `knownPeers` setting